pkg/knode-manager/utils/util.go

```diff
@@ -1 +1,10 @@
 package utils
+
```

Comment:
utils/util.go ? seems strange

Reply:
refer to this:
https://github.com/kubernetes/kubernetes/blob/master/pkg/util/config/config.go
https://github.com/kubernetes/kubernetes/blob/master/pkg/util/coverage/coverage.go

xxx/xxx.go

Reply:
sigs.k8s.io/kustomize/kyaml/internal/forked/github.com/qri-io/starlib/util/util.go

util/util.go

Reply:
They are not same.
 when they have `xxx/xxx.go`， there are only `xxx.go`  under the `xxx` package . But we have others

Reply:
this is not the key issue, don`t worry.